### ISSUE TITLE
Regression: current version no longer appears when upgrade pending

### ISSUE THREAD

**Issue by shi-bmz** [piwik/piwik#8369]
Hi,
When I went to upgrade to the latest version (from 2.13.1 to 2.14.1) I noticed that the current version again does not appear when I hover over the message.  This appears to be a regression of issue #2826 (Show current version when upgrade pending).

Thanks,
Shi.

**Comment by tsteur** [comment]
> I noticed that the current version again does not appear when I hover over the message. 

Where does the current version not appear? Sorry, I'm a little confused right now. At which step of the upgrading process are you?

**Comment by shi-bmz** [comment]
Current version: 2.13.1


What I see in the Dashboard and All Websites views:

NEW UPDATE: PIWIK 2.14.1

What I see when I hover the above message box:

Piwik 2.14.1 is available.
Please update now! (see changes).


What I see in the Administration view:

NEW UPDATE: PIWIK 2.14.1

What I see when I hover the above message box:

Piwik 2.14.1 is available.
Please update now! (see changes).
  CHECK FOR UPDATES
You are currently using Piwik 2.13.1.


For consistency, the Dashboard and All Websites views should show the 
same hover message as that shown in the Administration view (when the 
user has the appropriate permissions, of course)

Thanks,
Shi.

**Comment by tsteur** [comment]
:+1:

**Comment by mattab** [comment]
>  For consistency, the Dashboard and All Websites views should show the same hover message as that shown in the Administration view (when the user has the appropriate permissions, of course)

:+1:

**Comment by tsteur** [comment]
Just tried to reproduce and I'm seeing the current version in Dashboard and All Websites view. Also had a look at the code and cannot really see how this would not be shown. Which browser are you using?

**Comment by shi-bmz** [comment]
Original report from Firefox (recent version) on Windows 8.1
Currently using Firefox 12.0 on Linux (Ubuntu)

Currently, the Dashboard view shows no upgrade message (I did try upgrading to 2.14.1, but then that failed - I completed the installation by hand and noticed/read about some db data that got hosed so I restored from backup and reinstalled 2.13.1).

In Admin view, I see "Check for updates" and when I hover I see the expected message with version.

After clicking "Check for updates" in the hover box, I see a "NEW UPDATE" message box with the same hover content as before (except that now the new version is 2.14.2) - this message box also appears in the Dashboard view.
